reference-implementation/__tests__/resolving.js

```diff
@@ -169,12 +169,6 @@ describe('Mapped using the "imports" key only (no scopes)', () => {
       }
     }`);
 
-    it('should remap to built-in modules', () => {
-      expect(resolveUnderTest('/node_modules/als-polyfill/index.mjs')).toMatchURL('std:kv-storage');
-      expect(resolveUnderTest('https://example.com/node_modules/als-polyfill/index.mjs')).toMatchURL('std:kv-storage');
-      expect(resolveUnderTest('https://///example.com/node_modules/als-polyfill/index.mjs')).toMatchURL('std:kv-storage');
-    });
```

Comment:
Can we not keep this test, especially since it is implemented? Is this being removed because the resolver only knows about the `std:blank` builtin right now? Surely the list of supported builtins can be provided to the resolver initialization or as a host option?

Reply:
This test is fairly redundant with the newly-introduced ones, and yeah, it started failing once I made the implementation throw for anything not in `supportedBuiltInModules`. That will definitely be host-customizable, but for the reference implementation/tests, we don't need to make it so; the spec and tests will come out clear either way.